### ISSUE TITLE
↔ [Converter] ATen Softmax Intercept Function

### ISSUE THREAD

**Issue by gs-olive** [pytorch/TensorRT#2096]
# Context
Similarly to what exists for `einsum`, there should be a function to intercept `torch.nn.functional.softmax` calls pre-aot-autograd and replace it with the accelerated TRT implementation.

**Comment by gs-olive** [comment]
Not needed - address in decompositions + recent Torch nighly versions
